packages/backend/server/src/core/doc-renderer/__tests__/controller.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { randomUUID } from 'node:crypto';
+
+import { User, Workspace } from '@prisma/client';
+import ava, { TestFn } from 'ava';
+import request from 'supertest';
+import { Doc as YDoc } from 'yjs';
+
+import { createTestingApp, type TestingApp } from '../../../__tests__/utils';
```

Comment:
?

Reply:
The utils tools for unit testing are currently in the `src/__tests__` directory, let's reuse them for now 😭

Reply:
damn